spec/models/vacancy_spec.rb

```diff
@@ -382,12 +383,23 @@
     end
   end
 
-  describe 'delegate school_name' do
-    it 'should return the school name for the vacancy' do
-      school = create(:school, name: 'St James School')
-      vacancy = create(:vacancy, school: school)
+  describe '#school_or_school_group_name' do
```

Comment:
can we have the same for the method `#school_or_school_group_geolocation` which you've used?

Reply:
You could, but it would always be nil at the minute since there is no `geolocation` method on `SchoolGroup`

Reply:
Also, there was no test for the delegate geolocation in the first place